lib/flipper/adapters/moneta.rb

```diff
@@ -0,0 +1,122 @@
+require 'moneta'
+
+module Flipper
+  module Adapters
+    class Moneta
+      include ::Flipper::Adapter
+
+      FEATURES_KEY = :flipper_features
+
+      # Public: The name of the adapter.
+      attr_reader :name
+
+      # Public
+      def initialize(moneta)
+        @moneta = moneta
+        @name = :moneta
+      end
+
+      # Public:  The set of known features
+      def features
+        moneta[FEATURES_KEY] || Set.new
+      end
+
+      # Public: Adds a feature to the set of known features.
+      def add(feature)
+        moneta[FEATURES_KEY] = features << feature.key.to_s
+        true
+      end
+
+      # Public: Removes a feature from the set of known features and clears
+      # all the values for the feature.
+      def remove(feature)
+        moneta[FEATURES_KEY] = features.delete(feature.key.to_s)
+        moneta[key(feature.key)] = default_config
+        true
+      end
+
+      # Public: Clears all the gate values for a feature.
+      def clear(feature)
+        moneta[key(feature.key)] = default_config
+        true
+      end
+
+      # Public: Gets the values for all gates for a given feature.
+      #
+      # Returns a Hash of Flipper::Gate#key => value.
+      def get(feature)
+        default_config.merge(moneta[key(feature.key)].to_h)
+      end
+
+      # Public
+      def get_multi(features)
```

Comment:
*I so wish that moneta had bulk reads. Maybe we can try adding that someday upstream. *
yes that would be awesome and a fun feature to implement. Would also be sweet if  Moneta had instrumentation support like Flipper.  They have it for ActiveSupport::Cache, but not all adapters

You're totally right this can just use those methods